command/server.go

```diff
@@ -186,6 +189,13 @@ func (c *ServerCommand) Flags() *FlagSets {
 		Usage:   "Address to bind to in \"dev\" mode.",
 	})
 
+	f.BoolVar(&BoolVar{
+		Name:    "ignore-migration",
```

Comment:
Is this just here for testing? It seems like you'd never want to actually do this.

Reply:
Will be removed.